pkg/hostagent/group_assign.go

```diff
@@ -186,9 +186,16 @@ func (agent *HostAgent) assignGroups(pod *v1.Pod) (metadata.OpflexGroup, []metad
 	var namespace *v1.Namespace
 	if exists && namespaceobj != nil {
 		namespace = namespaceobj.(*v1.Namespace)
-
-		decodeAnnotation(namespace.ObjectMeta.Annotations[metadata.EgAnnotation], &egval, logger, "namespace[EpgAnnotation]")
-		decodeAnnotation(namespace.ObjectMeta.Annotations[metadata.SgAnnotation], &sgval, logger, "namespace[SgAnnotation]")
+                var temp metadata.OpflexGroup
+                decodeAnnotation(namespace.ObjectMeta.Annotations[metadata.EgAnnotation], &temp, logger, "namespace[EpgAnnotation]")
```

Comment:
Please make the vmmType check the first level check so it's clear it's applicable to only openshift. Also, please add a comment.

Reply:
Added, thanks!